src/components/appointments/LocationAndTimeStep.spec.js

```diff
@@ -60,27 +69,29 @@ beforeEach(() => {
 afterEach(() => {
   jest.runOnlyPendingTimers();
   jest.useRealTimers();
+  window.sessionStorage.clear();
 });
 
 describe('The location and time step', () => {
   it('disables date and time until a location is selected', async () => {
     mswServer.use(mockAppointmentProductsGet, mockAppointmentLocationsGet);
 
-    render(
-      <LocationAndTimeStep />,
-      // product with multiple locations, see ./mocks.js
-      {
-        products: [{productId: 'e8e045ab', amount: 1}],
-        location: '',
-        date: '',
-        datetime: '',
-      }
-    );
-
-    // No location should be selected, as there are multiple options
-    await waitFor(() => {
-      expect(screen.queryByText('Open Gem')).not.toBeInTheDocument();
-      expect(screen.queryByText('Bahamas')).not.toBeInTheDocument();
+    await act(async () => {
+      render(
+        <LocationAndTimeStep />,
+        // product with multiple locations, see ./mocks.js
+        {
+          products: [{productId: 'e8e045ab', amount: 1}],
+          location: '',
+          date: '',
+          datetime: '',
+        }
+      );
+      // No location should be selected, as there are multiple options
+      await waitFor(() => {
+        expect(screen.queryByText('Open Gem')).not.toBeInTheDocument();
+        expect(screen.queryByText('Bahamas')).not.toBeInTheDocument();
+      });
```

Comment:
```suggestion
    await act(async () => {
      render(
        <LocationAndTimeStep />,
        // product with multiple locations, see ./mocks.js
        {
          products: [{productId: 'e8e045ab', amount: 1}],
          location: '',
          date: '',
          datetime: '',
        }
      );
      // No location should be selected, as there are multiple options
    });
    await waitFor(() => {
      expect(screen.queryByText('Open Gem')).not.toBeInTheDocument();
    });
    expect(screen.queryByText('Bahamas')).not.toBeInTheDocument();

```
https://github.com/testing-library/eslint-plugin-testing-library/blob/ba14becb6b61d80cc029ddf9e249ba5e19020ffe/docs/rules/no-wait-for-multiple-assertions.md#rule-details

Reply:
And there's another one of those in:
`autoselects location when there is only one option and enables the date field`

Reply:
I thought I tried this variant and wasn't actually able to get rid of the warning about act, did that work for you?